packages/design-system/src/styles/theme/oc-text.scss

```diff
@@ -18,6 +18,10 @@ html {
   font-size: var(--oc-font-size-default);
 }
 
+button {
```

Comment:
Please do it like this:

```css
html, button {
  font-feature-settings: "cv11";
}

html {
  font-size: var(--oc-font-size-default);
}
```

Otherwise you would duplicate the cv11 setting.